content/rust/concepts/strings/strings.md

```diff
@@ -0,0 +1,146 @@
+---
+Title: 'Strings'
+Description: 'This entry covers creating, manipulating and referencing strings (encodings of UTF-8 sequences) in Rust.'
```

Comment:
Describe what are strings in Rust rather than using this sentence. Write a one-line description.